packages/dev/core/src/Materials/effect.ts

```diff
@@ -460,6 +460,12 @@ export class Effect implements IDisposable {
         return this._engine;
     }
 
+    /**
+     * The pipeline context for this effect
+     * @returns the associated pipeline context
+     */
+    public getPipelineContext<T extends IPipelineContext>(): Nullable<T>;

```

Comment:
Let s not add this one. The type changes at runtime depending on the running environment.